app/src/main/java/org/mozilla/focus/widget/Cursor.java

```diff
@@ -71,28 +77,44 @@ public float getSpeed() {
 
     public void moveCursor(Direction direction) {
         activeDirections.add(direction);
-        
+
+        // If the cursor isn't moving start the move loop
```

Comment:
nit: I think you could do without this comment if you changed `handler` -> `animationFrameHandler`